src/main/resources/application.yml

```diff
@@ -0,0 +1,9 @@
+server:
+  port: 8080
+openweathermap:
+      api:
+           key: 675bf40b4d52b5fb5e47199ec8315931
+           url: https://api.openweathermap.org/data/2.5/weather
+
+connectionTimeout: 3000
+readTimeout: 3000
```

Comment:
please replace to:
   `restTemplate:`
          `properties:`
              `connectionTimeout: 3000`
    `readTimeout: 3000`